src/gd_kotlin.cpp

```diff
@@ -393,9 +393,9 @@ void GDKotlin::unregister_classes(jni::Env& p_env, jni::JObjectArray p_classes)
 
 KtClass* GDKotlin::find_class(const StringName& p_script_path) {
 #ifdef DEBUG_ENABLED
-    //FIXME: https://github.com/utopia-rise/godot-jvm/issues/76
-    JVM_ERR_FAIL_COND_V_MSG(!classes.has(p_script_path), nullptr,
-                   vformat("Failed to find class for path: %s", p_script_path));
+    if (!classes.has(p_script_path)) {
```

Comment:
I would at least make a print verbose here or even a print WARN.

Reply:
Not sure about that. Because this would print that message or warning for each kt class which is not registered in godot.
Which leads to hundret's of prints for bigger projects.

Reply:
This would clutter up the output in such a great way that it renders it unusable (if we use warnings) and mostly unusable for verbose logs in general

Reply:
True, anyway I don't have strong opinion, we'll see if we need it later for debugging.

Reply:
:+1: